README.md

```diff
@@ -1,2 +1,62 @@
-# catalog-of-my--hings-app
-"Catalog of my things" is  a simple console app that allows users to manage collections of the things they own.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CATALOG OF MY THINGS APP
+
+> "Catalog of my things" is  a simple console app that allows users to manage collections of the things they own.
+
+Additional description about the project and its features.
+## Built With
+- ruby
+- rubocop
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VScode, Atom, Sublime text, etc)
+- Basic knowledge of ruby
+
+### Setup
+
+- clone repository using `git clone`
+- git clone `https://github.com/kinginthenorthcodez/catalog-of-my-things-app`
+- cd `catalog-of-my-things-app`.
+
```

Comment:
[OPTIONAL] It would be great to add some instructions about how to run the app like `ruby app/main.rb`